api-spec/sftt/api-spec.md

```diff
@@ -205,14 +206,84 @@ Invite someone to join a team. Will send an email to all specified people that i
 }
 ```
 
-## `POST /teams/:team_id/join`
+## `POST /teams/:team_id/apply`
 
-Join this team. Any member that is not currently a part of a team can join any team.
+Apply to join this team. Any member can apply to join a team that they are not currently on. They will have to be approved by the team leader before becoming an actual member of the team.
 
 ### Request
 
 No request body.
 
+## `GET /teams/:team_id/applicants`
```

Comment:
Where would the team leader see this? Below the current members, in the same format? Or before that, so it doesn't fall off the screen if there's a lot of members?

Reply:
Before, show a notification and only show applicants when they click on it